packages/rainbowkit/src/components/AccountModal/AccountModal.tsx

```diff
@@ -13,10 +13,11 @@ export interface AccountModalProps {
 
 export function AccountModal({ onClose, open }: AccountModalProps) {
   const { address } = useAccount();
-  const { data: balanceData } = useBalance({ address });
   const ensName = useMainnetEnsName(address);
   const ensAvatar = useMainnetEnsAvatar(ensName);
   const { disconnect } = useDisconnect();
+  const [balanceData, setBalanceData] =
+    useState<ReturnType<typeof useBalance>['data']>();
```

Comment:
Not sure if it makes sense to do another PR for this, but the chain modal would also call `"eth_getBalance"` randomly. I made sure to only call `"eth_getBalance"` if the chain modal is open.